packages/xo-server/src/xapi/mixins/vm.mjs

```diff
@@ -490,10 +490,15 @@ const methods = {
 
     hasVendorDevice: true,
 
+    // deprecated, property kept for backward compatibility, for XCP 8.3+ use nestedVirt instead
```

Comment:
This is not deprecated.
If you are on XCP 8.2, you have to use this field.

The deprecated flag is only for reading the value `expNestedHvm` from an `XO object`
````suggestion
    // for XCP 8.3+ use nestedVirt instead
```